docs/up-ga/sessions/v1.md

```diff
@@ -9,10 +9,66 @@
 # Up-ga Sessions Measure
 
 <div id="value-display"></div>
+<br>
 <strong>Description:</strong>
 <div class="tile-1" style="text-align:center; color:black">
-Total sessions for the landing page of a work, as determined by Google Analytics. A Session is a group of visits made by the same user within a continuous time frame, and ends after 30 minutes of inactivity; if a single user keeps interacting with the website within this time frame, multiple visits to the same work will be counted as one session.
+Total views of the landing page of a work, as determined by Google Analytics (i.e. webserver logs) of the website. To count multiple views of the same landing page by the same user, there must be a gap of at least 30 minutes from the last view of that landing page, even if that previous view was not counted.
```

Comment:
I think it's best to remove "i.e. webserver logs" - Google Analytics and webserver logs are two different things. Google Analytics uses JavaScript to measure page visits and page interactions.